tests/legacy/field-props.test.tsx

```
@@ -55,22 +55,38 @@ describe('legacy.field-props', () => {
-            <Input />
-          </Field>
-        </Form>
-      </div>,
-    );
+  describe('normalize', () => {
+    it('basic', async () => {
+      const form = React.createRef<FormInstance>();
+      const { container } = render(
+        <div>
+          <Form ref={form}>
+            <Field name="normal" normalize={v => v && v.toUpperCase()}>
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)